cvat-core/src/server-proxy.js

```diff
@@ -618,9 +634,97 @@
                     throw generateError(errorData);
                 }
 
-                onUpdate('The data are being uploaded to the server..');
+                onUpdate('The data are being uploaded to the server 0%');
+
+                async function chunkUpload(taskId, file) {
+                    const initChunkUpload = await Axios.post(`${backendAPI}/tasks/${response.data.id}/data`,
+                        taskData, {
+                            proxy: config.proxy,
+                            params: {
+                                action: 'init-chunk-upload',
```

Comment:
@klakhov , why here `init-chunk-upload` action is a part of body and `submit` action is a query parameter?